platform/admin/modules/admin-security-ui/pom.xml

```diff
@@ -265,28 +265,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-antrun-plugin</artifactId>
-            </plugin>
-            <!--<plugin>-->
```

Comment:
I realize it's a WIP; but eventually we can just remove this commented plugin

Reply:
Actually I already removed it 😄